packages/sanity/src/core/releases/tool/detail/ReleaseSummary.tsx

```diff
@@ -75,14 +74,38 @@ export function ReleaseSummary(props: ReleaseSummaryProps) {
 
   const filterRows = useCallback(
     (data: DocumentInRelease[], searchTerm: string) =>
-      data.filter(({previewValues, isPending}) => {
+      data.filter(({document}) => {
+        // this is a temporary way of doing the search without the previews
```

Comment:
would be great if this could be extracted and unit tested in isolation, if time allows.